plugins/inputs/zfs/zfs_linux.go

```diff
@@ -83,9 +83,9 @@ func getTags(pools []poolInfo) map[string]string {
 	return map[string]string{"pools": poolNames}
 }
 
-func gather(lines []string, fileLines int) ([]string, []string, error) {
-	if len(lines) != fileLines {
-		return nil, nil, errors.New("expected lines in kstat does not match")
+func gather(lines []string, minimalFileLines int) ([]string, []string, error) {
+	if len(lines) < minimalFileLines {
+		return nil, nil, errors.New("lines in kstat is not enough")
```

Comment:
Can you please minimize the changes here by keeping the parameter-name `fileLines` instead of renaming it!?! The same holds true for the error message.